est/004.est

```diff
@@ -0,0 +1,24 @@
+date: 13-8-2016
+author: Mihai Andronache
+method: champions.pert
+scope:
+  1: Database layer changes
+  2: User actions implementation and @help user
+  3: UI fixes (colors, broken actions etc)
+  4: Bug fixes and refactorings (fix reported issues)
+  5: Unit/Integration testing
+  6: Infrastructure fixes (CI and build related fixes)
+
+champions:
+  1:
+    worst-case: 32
+    best-case: 16
+    most-likely: 24
```

Comment:
@amihaiemil here champions should be sorted from descending using `most-likely` field